src/AlgebraicGeometry/Surfaces/EllipticSurface/Morphisms.jl

```diff
@@ -94,12 +94,10 @@ end
 end
 
 
-function raw_reduction_of_algebraic_lattice(X::EllipticSurface)
-  return get_attribute!(X, :raw_reduction_of_algebraic_lattice) do
-    X_red_raw, bc = raw_good_reduction(X)
-    basis_ambient, _, _= algebraic_lattice(X)
-    return red_dict = IdDict{AbsWeilDivisor, AbsWeilDivisor}(D=>_reduce_as_prime_divisor(bc, D) for D in basis_ambient)
-  end::IdDict
+@attr IdDict function raw_reduction_of_algebraic_lattice(X::EllipticSurface)
```

Comment:
While we are at it, might as well make a few more minor improvements :-)
```suggestion
@attr IdDict{AbsWeilDivisor, AbsWeilDivisor} function raw_reduction_of_algebraic_lattice(X::EllipticSurface)
```